File: IDATT2106 Systemutvikling 2 med smidig prosjekt/backend/src/test/java/com/ntnu/gidd/utils/JpaUtils.java
package com.ntnu.gidd.utils;

import com.querydsl.core.types.Predicate;
import com.querydsl.core.types.dsl.Expressions;
import org.springframework.data.domain.PageRequest;
import org.springframework.data.domain.Pageable;

public class JpaUtils {

    public static Predicate getEmptyPredicate() {
        return Expressions.asBoolean(true).isTrue();
    }

    public static Pageable getDefaultPageable() {
        return PageRequest.of(0, 25);
    }
}

File: IDATT2105 Full-stack applikasjonsutvikling/Rombestilling/backend/src/test/kotlin/ntnu/idatt2105/util/JpaUtils.kt
package ntnu.idatt2105.util

import com.querydsl.core.types.Predicate
import com.querydsl.core.types.dsl.Expressions
import org.springframework.data.domain.PageRequest
import org.springframework.data.domain.Pageable

class JpaUtils {

    fun getDefaultPageable(): Pageable {
        return PageRequest.of(0, 25)
    }

    fun getEmptyPredicate(): Predicate {
        return Expressions.asBoolean(true).isTrue
    }
}
